CONTRIBUTING.md

```diff
@@ -25,24 +25,25 @@ cd $GOPATH/src/go.uber.org/fx
 ## Dependency management
 
 Dependencies are tracked via `glide.yaml`. If you're not familiar with `glide`,
-read the [docs](https://github.com/Masterminds/glide#usage).
+[read the docs](https://github.com/Masterminds/glide#usage).
 
-## Licence headers
+## License headers
 
-This project is Open Source Software, and requires a header at the beginning of
-all source files. This is enforced by commit hooks and TravisCI.
+This project is open source software and requires a header at the beginning of
+every source file. This is enforced by commit hooks and TravisCI.
 
-To add licence headers, use
+To add license headers, use
 [uber-licence](https://github.com/uber/uber-licence):
```

Comment:
This is out of date, we don't use the npm version anymore.
We now use `go.uber.org/tools/update-license` and it comes as part of `make dependencies`